src/outputs/temp_file_to_be_deleted.py

```diff
@@ -0,0 +1,43 @@
+
+import pandas as pd
+
+from src.utils.wrappers import exception_wrap
+
+
+@exception_wrap
+def combine_dataframes(main_df: pd.DataFrame, mapper_csv_path: pd.DataFrame) -> pd.DataFrame:
+    """
+    Combine two DataFrames using a left join based on specified columns.
+
+    Args:
+        main_df (pandas.DataFrame): The main DataFrame.
+        mapper_csv_path (str): The path to the CSV file for the mapper DataFrame.
+
+    Returns:
+        pandas.DataFrame: The combined DataFrame resulting from the left join.
+    """
+    try:
+        # Read the mapper DataFrame from CSV
+        mapper_df = pd.read_csv(mapper_csv_path)
+        
+        # Perform left join
+        combined_df = main_df.merge(mapper_df, how='left', left_on='reference', right_on='ruref')
```

Comment:
This merge looks fine!

It should go in the outputs/outputs_main.py script. Also, if you could find a useful name instead of mapper_df that would be good.